src/neovim/nvimUtil.ts

```diff
@@ -9,11 +9,16 @@ import { spawn } from "child_process";
 import { attach } from "promised-neovim-client";
 import { Register, RegisterMode } from "../register/register";
 import { ModeName } from "../mode/mode";
+import * as fs from "fs";
+import * as os from "os";
+
+type UndoTree = {entries: Array<{seq: number, time: number}>, save_cur: number, save_last: number, seq_cur: number, seq_last: number, synced: number, time_cur: number};
 
 export class Neovim {
 
   static async initNvim(vimState: VimState) {
-    const proc = spawn(Configuration.neovimPath, ['-u', 'NONE', '-N', '--embed'], { cwd: __dirname });
+    this.syncVSSettingsToVim();
+    const proc = spawn(Configuration.neovimPath, ['-u', '~/.config/nvim/init.vim', '-N', '--embed'], { cwd: __dirname });
```

Comment:
I haven't added the logic yet, but I think the neovim embedding in VSCode should use a custom init.vim file. We're going to need to do a lot of autogeneration of the init.vim file for various setting sync reasons, and I'd rather not be overwriting users' original settings.